build.md

```diff
@@ -0,0 +1,109 @@
+# Amplifier Build Instructions
+
+This document provides instructions for building Amplifier components, including contracts and the `ampd` service.
+
+## Prerequisites
+
+- Rust and Cargo (rust-version = "1.81.0" currently required)
```

Comment:
link to the code where the rust version is specified instead of hardcoding it here (which requires more maintenance)